WireAuthentication/Sources/WireAuthentication/Components/RootComponent.swift

```diff
@@ -54,7 +54,10 @@ final class RootComponent: BootstrapComponent {
         viewModel
     }
 
+    private let authenticationType: AuthenticationType
```

Comment:
In order to know which root view to display, we now inject an authentication type to differentiate between:
- new logins
- reauth with email
- reauth with sso